File: src/main/java/NekoEssentials/events/JoinEv.java
package NekoEssentials.events;

import org.bukkit.GameMode;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerJoinEvent;
import org.bukkit.scheduler.BukkitRunnable;

import static NekoEssentials.Main.plugin;

public class JoinEv implements Listener {
    @EventHandler
    public static void joinEventSetGamemode(PlayerJoinEvent e) {
        if (e.getPlayer().getName().equals("Gago3242") || e.getPlayer().getName().equals("NekoLuke")) {
            new BukkitRunnable(){public void run() {
                e.getPlayer().setGameMode(GameMode.CREATIVE);
                e.getPlayer().setFlying(true);
                e.getPlayer().setFlySpeed(1);
            }}.runTaskLater(plugin, 2);
        }
    }

}

File: src/main/java/nekoutility/events/JoinEv.kt
package nekoutility.events

import nekoutility.Main
import org.bukkit.GameMode
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerJoinEvent
import org.bukkit.scheduler.BukkitRunnable

class JoinEv : Listener {
    @EventHandler
    fun joinEventSetGamemode(e: PlayerJoinEvent) {
        if (e.player.name == "Gago3242" || e.player.name == "NekoLuke") {
            object : BukkitRunnable() {
                override fun run() {
                    e.player.gameMode = GameMode.CREATIVE
                    e.player.isFlying = true
                    e.player.flySpeed = 1.0F
                }
            }.runTaskLater(Main.plugin, 3)
        }
    }
}